enterprise/metrics/src/main/java/org/neo4j/metrics/source/JvmMetrics.java

```diff
@@ -37,10 +37,10 @@
 
 import static com.codahale.metrics.MetricRegistry.name;
 
-@Documented( ".Java Virtual Machine Metrics ++\n" +
-             "These are environment dependent and they may vary on different hardware and with JVM configurations.\n" +
+@Documented( "=== Java Virtual Machine Metrics\n\n" +
```

Comment:
@davidegrohmann Because there's no table after the title in this case. If we used a separate `@Title` annotation we could have a cleaner solution for this.